holidays/financial/bolsas_y_mercados_espanoles.py

```
@@ -0,0 +1,96 @@
+        [2024](https://web.archive.org/web/20240530063923/https://www.bolsasymercados.es/bme-exchange/es/Negociar/Calendario-Mercado)
+        [2025](https://web.archive.org/web/20250804221451/https://www.bolsasymercados.es/bme-exchange/es/Negociar/Calendario-Mercado)
+        [2026](https://web.archive.org/web/20260217165615/https://www.bolsasymercados.es/bme-exchange/es/Negociar/Calendario-Mercado)
+    """
+
+    market = "XMAD"
+    default_language = "es"
+    supported_languages = ("en_US", "es")
+    start_year = 2000
+    supported_categories = (HALF_DAY, PUBLIC)
+
+    def __init__(self, *args, **kwargs):
```

Ruff (0.15.21)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)